Add post on April tax reforms (#2451)
PolicyEngine/policyengine-app: src/posts/articles/impact-of-tax-changes-2025-2026.md

Four tax reforms will take effect in April 2025. Using the PolicyEngine UK tax-benefit microsimulation model, we analysed these reforms, which will increase taxes by an average of £1,112 per household.

This analysis, requested and [published](https://insider.iea.org.uk/p/new-analysis-the-cost-of-tax-hikes) by the Institute for Economic Affairs, featured in the Daily Mail's [front-page article](https://www.dailymail.co.uk/news/article-14548907/Labours-tax-raids-cost-families-week.html) on the 28th March 2025, in two [Express](https://www.express.co.uk/news/politics/2034186/rachel-reeves-tax-increase-cost) [articles](https://www.express.co.uk/finance/personalfinance/2034211/rachel-reevesl-tax-hikes-cost-british-families), and two [other](https://www.dorsetecho.co.uk/news/25047728.tax-hikes-will-cost-british-families-1-000-per-year-average/) [outlets](https://londonlovesbusiness.com/labours-tax-hikes-will-cost-british-families-over-1000-per-year-on-average/).

## Policy changes

> From a combination of temporary reforms expiring and new tax reforms taking effect

Reviewer: Can you also please specify when these reforms were announced in more detail, like in the third bullet? (I'd genuinely like to know without looking it up)
```suggestion
From a combination of temporary reforms expiring and new tax reforms taking effect:
```

Four tax reforms will take effect in April 2025. Using the PolicyEngine UK tax-benefit microsimulation model, we analysed these reforms, which will increase taxes by an average of £1,112 per household.

This analysis, requested and [published](https://insider.iea.org.uk/p/new-analysis-the-cost-of-tax-hikes) by the Institute for Economic Affairs, featured in the Daily Mail's [front-page article](https://www.dailymail.co.uk/news/article-14548907/Labours-tax-raids-cost-families-week.html) on the 28th March 2025, in two [Express](https://www.express.co.uk/news/politics/2034186/rachel-reeves-tax-increase-cost) [articles](https://www.express.co.uk/finance/personalfinance/2034211/rachel-reevesl-tax-hikes-cost-british-families), and two [other](https://www.dorsetecho.co.uk/news/25047728.tax-hikes-will-cost-british-families-1-000-per-year-average/) [outlets](https://londonlovesbusiness.com/labours-tax-hikes-will-cost-british-families-over-1000-per-year-on-average/).

## Policy changes

> The April 2025 tax reforms introduce several changes to the UK tax system:

Reviewer: who instituted them? trace back to the government action and make active voice, e.g. "In its 2024 Autumn Budget, the Labour Government instituted these changes effective 1st April 2025:

- Lowering the Stamp Duty Land Tax nil-rate threshold...

Author: OK added active voice and adding legislation links for all but council tax

- Stamp Duty Land Tax nil-rate thresholds will be set at £300,000 (from £425,000) for first-time buyers and £125,000 (from £250,000) for subsequent purchases. These thresholds determine the property value at which SDLT begins to apply. First-time buyers maintain a higher threshold than subsequent purchasers under this reform.

> - Employer's National Insurance rate will change to 15% with the secondary threshold set to £96.14 (from £175) per week. We assume that the employer NI changes will be passed on to employees at 40% in 2025, 50% in 2026, 60% in 2027, and 70% in 2028-2029, with the remainder split between business owners and prices.

Author: Linked

        "size": 16
      },
      "x": 0,
      "xanchor": "left"
    },
    "yaxis": {
      "title": "Median annual tax rise (£)",
      "titlefont": {
        "family": "Roboto Serif"
      },
      "tickfont": {
        "family": "Roboto Serif"
      },
      "automargin": true
    },
    "xaxis": {
      "title": "Region",
      "titlefont": {
        "family": "Roboto Serif"
      },
      "tickangle": 45
    },
    "height": 500,
    "margin": {
      "l": 50,
      "r": 50,
      "b": 150,
      "t": 100,
      "pad": 4
    },
    "annotations": [
      {
        "x": 1,
        "y": -0.4,
        "xref": "paper",
        "yref": "paper",
        "text": "Source: PolicyEngine UK",
        "showarrow": false,
        "font": {
          "family": "Roboto Serif",
          "size": 10,
          "color": "#616161"
        }
      }
    ],
    "images": [
      {
        "source": "/logo512.png",
        "x": 1,
        "y": -0.35,
        "xref": "paper",
        "yref": "paper",
        "sizex": 0.1,
        "sizey": 0.1,
        "xanchor": "right",
        "yanchor": "bottom"
      }
    ],
    "plot_bgcolor": "white",
    "paper_bgcolor": "white"
  }
}
```

## Conclusion

> These reforms represent a significant change to the UK tax system, with projected revenue of over £31 billion in 2025/26. The distributional analysis shows that while all income groups face increased tax burdens, the monetary impact is strongly progressive, with the highest income households facing the largest increases.

Reviewer: ```suggestion
We project that these reforms will raise £31 billion in 2025/26, with the highest income households facing the largest increases.
```

Author: De-claudified

## Policy changes

From a combination of temporary reforms expiring and new tax reforms taking effect:

> - The Government will [set](https://www.legislation.gov.uk/ukpga/2023/2/section/1) **Stamp Duty Land Tax** nil-rate thresholds at £300,000 (from £425,000) for first-time buyers and £125,000 (from £250,000) for subsequent purchases. These thresholds determine the property value at which SDLT begins to apply. First-time buyers maintain a higher threshold than subsequent purchasers under this reform. This reform is the result of an expiry of a Stamp Duty Land Tax cut announced by then-Chancellor Jeremy Hunt in the Autumn Budget 2023 (expiring 31 March 2025).

Reviewer: ok...i wouldn't really say the government is setting the rate if they're just allowing a previously scheduled expiration to occur (per our current law/current policy discussion this morning). can we describe these as tax changes instead?

Author: Yes sure though no active voice then

Author: Actually amended- still active
